export_defaults.sh

```
@@ -0,0 +1,98 @@
+      # Clean up the key and value
+      key=$(echo "$key" | sed 's/"//g')
+      
+      echo "defaults write \"$domain\" \"$key\" $value" >> "$DEFAULTS_FILE"
+    done
+  fi
+done
+
+# Process remaining domains
+for domain in "${domains[@]}"; do
+  # Skip domains we've already processed
+  if [[ " ${common_domains[@]} " =~ " ${domain} " ]]; then
```

Shellcheck (0.10.0)

[error] Arrays implicitly concatenate in [[ ]]. Use a loop (or explicit * instead of @).

(SC2199)